syntax = "proto3";

package jio.brain.proto.knowledge.api.data;

option java_package = "com.jio.brain.proto.knowledge.api.data";
option java_multiple_files = true;

import "jio/brain/proto/base/status.proto";

service GetContextValueNameService {
  rpc serve(GetContextValueNameRequest) returns (GetContextValueNameResponse) {}
}

message GetContextValueNameRequest {
  uint32 context_id = 1;
  uint32 value_id = 2;
}

message GetContextValueNameResponse {
  string context_value_name = 1;
  jio.brain.proto.base.BrainStatus status = 2;
}

